tests/js/ttimes.nim

```diff
@@ -9,21 +8,32 @@ import times
 # Tue 19 Jan 03:14:07 GMT 2038
 
 block yeardayTest:
-  # check if yearday attribute is properly set on TimeInfo creation
-  doAssert fromSeconds(2147483647).getGMTime().yearday == 18
-
-block localTimezoneTest:
-  # check if timezone is properly set during Time to TimeInfo conversion
-  doAssert fromSeconds(2147483647).getLocalTime().timezone == getTimezone()
```

Comment:
Why were these tests removed?

Reply:
The test assumes that whatever timezone offset was active at `2147483647` is the same as the one currently used (the one returned by `getTimezone()`). That might not be the case, so the test doesn't work.